userguide/jails.rst

```diff
@@ -488,6 +489,16 @@ of these options.
    +--------------------------+--------------+---------------------------------------------------------------------------------------------------------------------+
 
 
+.. warning::
+   :name: security warning
+
+   :guilabel:`allow_raw_sockets` enables Inernet Control Message
+   Protocol (ICMP) and
+   `tcpdump <https://en.wikipedia.org/wiki/Tcpdump>`__
+   in the jail. This allows access to the raw sockets of the host and
+   poses a security threat.
```

Comment:
s/poses/can pose/, but "pose" is a strange verb to use.  How about "can be a security hazard."?
We should also link to something that describes raw sockets more, like https://en.wikipedia.org/wiki/Network_socket#Datagram_socket.